package.json

```diff
@@ -37,5 +37,8 @@
   "bugs": {
     "url": "https://github.com/katzer/cordova-plugin-background-mode/issues"
   },
-  "homepage": "https://github.com/katzer/cordova-plugin-background-mode#readme"
+  "homepage": "https://github.com/katzer/cordova-plugin-background-mode#readme",
```

Comment:
should we change the homepage URL?

Reply:
I'm going to make a PR of `isBatteryOptimizationsDisabled()` to the official repo. This repo is temporary until it's merged.